tp/tp.py

```diff
@@ -136,6 +138,21 @@ def refresh(self):
                              self.previous_as_count, self.managed_by_autoscale())
             self.previous_as_count = self.managed_by_autoscale()
 
+    def is_ec2_state_running(self, instance_id):
+        def ec2_state(i_id):
+            return self.ec2.get_all_instance_status(instance_ids=[i_id])
+
+        try:
+            found_instance = ec2_state(instance_id)
+            return len(found_instance) > 0 and found_instance[0].state_name not in (
+                'terminated', 'shutting-down')
```

Comment:
Currently, the possibles states are `pending | running | shutting-down | terminated | stopping | stopped`. Since we wait for the instance to be running before attaching tit to the ELB, I agree with you. Will check for running instead.